src/apis/mod.rs

```diff
@@ -16,7 +16,7 @@ pub enum Error<T> {
     ResponseError(ResponseContent<T>),
 }
 
-impl <T> fmt::Display for Error<T> {
+impl<T> fmt::Display for Error<T> {
```

Comment:
I would suggest to remove all this error handling code and use something like the `thiserror` crate to automatically handle everything.

You can see that in action inside of sigstore-rs and oci-distribution